en/guides/codablerouting.md

```diff
@@ -15,172 +15,259 @@ redirect_from:
 
 <div class="titleBlock">
   <h1>Codable Routing</h1>
-  <p>How to use Codable Routing in Kitura</p>
 </div>
 
 
 
 ## Introduction
-Kitura 2.0 introduces “Codable Routing”. Here the router handlers are much like normal functions you might define elsewhere in your code: they take struct or class types as parameters, and respond with stuct or class types via a completion handler, with just an additional requirement that those types implement [Codable](https://developer.apple.com/documentation/swift/codable) from Swift 4 (hence the name).
+Kitura has two types of routing. Raw Routing, which works directly with Router Requests and Router Responses, and Codable Routing which works directly with Swift types, provided they conform to [Codable](https://developer.apple.com/documentation/swift/codable). 
 
-Codable Routing isn’t suitable for every use case and scenario, so Raw Routing is still available where you need the power and flexibility, but it is perfect for building REST APIs such as you might want to do to build a Backend For Frontend (BFF) for an iOS app.
+Codable Routing was made available in Kitura 2.0, before this release only Raw Routing was available. Kitura 2.0 supports both types of Routing, depending on what your application needs. You can implement both types of routing in your application as well. This offers a lot of flexibility on how you build your server. 
 
-## Building a Create API for POST requests
+For many, Codable Routing feels more 'natural' as you're getting to work directly with Swift types.  
+Over the next sections we will go through step by step how to set up Codable routes on a Kitura server.  
```

Comment:
I don't like the "Over the next sections" wording as looking down there are only one or two sections.  I don't know if you intended to split this into more sections? 
How about "This guide will show you step by step how to set up Codable routes on a Kitura server."

Reply:
You are correct, there is more sections to come but for now I think you're right it doesn't make sense so I'll update that :). 

The `So...` was intended to be a leading statement, to help transition to the Demonstration section but I can remove it if it doesn't read well :)